independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/model/HasPriority.java

```diff
@@ -14,6 +14,13 @@ class TreeMapComparator implements Comparator<HasPriority> {
 
         public static final TreeMapComparator INSTANCE = new TreeMapComparator();
 
+        public static final TreeMapComparator REVERSED = new TreeMapComparator() {
```

Comment:
This can be simplified with `Collections.reversedOrder(INSTANCE)` or by doing the reverse call in the caller

Reply:
I actually like this idiom here because it follows the same principle as in the rest of RESTEasy Reactive codebase